runbot_docker_orchest/models/runbot_repo.py

```diff
@@ -0,0 +1,69 @@
+from odoo import models, fields, api
+from dateutil import parser
+
+
+class RunbotRepo(models.Model):
+    _inherit = "runbot.repo"
+
+    is_quayio_image = fields.Boolean('Use quay.io image')
+    quayio_repo_name = fields.Char("Quay.io repo")
+
+    def create_build_orchest(self):
+        """Method create a build for image of orchestv
+        """
+        repo = self
+        settings = self.env['res.config.settings'].setting_quayio()
+        build_obj = self.env['runbot.build']
+        branch_obj = self.env['runbot.branch']
+        image_tags = settings.get_quayio_tags(
```

Comment:
Add a `if` to skip if is repo.hook == True like as:
https://github.com/odoo/runbot/blob/4a558741e01b206152ed10f5062d0fc94121fed5/runbot/models/repo.py#L140

And save a file with the last docker pull command executed in order to compare it with repo.hook_time